File: R/03_eda/correlationAnalysis/correlationAnalysis.jl
# Importar el conjunto de datos diamonds
using RDatasets
diamonds = dataset("ggplot2", "diamonds")

# Calcular los coeficientes de correlación
using Statistics
Statistics.cor(diamonds[:, 1], diamonds[:,5])
Statistics.cor(diamonds[:, 1], diamonds[:,6])
Statistics.cor(diamonds[:, 1], diamonds[:,7])

# Visualizar los resultados
using Plots
scatter(diamonds[:, 1], diamonds[:, 7], xlabel = "Carat", ylabel = "Price")

File: R/03_eda/correlationAnalysis/correlationAnalysis.py
# Importar el conjunto de datos diamonds
import seaborn as sns
diamonds = sns.load_dataset('diamonds')

# Calcular los coeficientes de correlación
diamonds[['carat', 'depth', 'table', 'price']].corr()

# Calcular la matriz de correlación
diamonds[['carat', 'depth', 'table', 'price']].corr(method='pearson')

# Visualizar los resultados
import matplotlib.pyplot as plt
plt.scatter(diamonds['carat'], diamonds['price'])
plt.xlabel('Carat')
plt.ylabel('Price')
plt.show()

